### PR TITLE
DRAFT: 1.40.0 changelog

### PR DESCRIPTION
**Pull request by spikecurtis**

### CODE REVIEW

**Review by code-asher — APPROVED** [review]

**Review by johnstcn — APPROVED** [review]

File: changelog/1.40.0.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# 1.40.0
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.40.0.
```
Comment (sreya) [inline changelog/1.40.0.md:5]: We should add something in bold that indicates all users must restart their workspaces after this update if they wish to access them.

Comment (johnstcn) [inline changelog/1.40.0.md:5]:
```suggestion
**After upgrading to this release, all workspaces will need to be rebuilt in order for users to access them.**
```

Comment (spikecurtis) [reply changelog/1.40.0.md:5]: Er, wait.  Why do they need to restart their workspaces?

File: changelog/1.40.0.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- Fixed an issue where deleted users would show up when using a filter in the
+  user list.
+- Fixed an issue where "create" audit logs would be emitted when existing users
+  log in over OIDC.
+- Fixed an issue where images and tags from a deleted organization would
+  incorrectly be retained in the database.
+- Fixed an issue where some audit log diffs showed `[object Object]` instead of
+  the changes.
+- Fixed an issue where Organization names were incorrectly required to be at
+  least 3 characters.
```
Comment (johnstcn) [inline changelog/1.40.0.md:34]:
```suggestion
  least 3 characters.
- Fixed an issue where updates to a workspace provider would not take
  effect until all `coderd` replicas were restarted.
```

**Review by johnstcn — APPROVED** [review]

**Pull request merged**